jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+---
+layout: classic-docs
+title: "Bundling Config"
+short-title: "Bundling Config"
+description: "Bundling configuration with CircleCI 2.1"
+categories: [configuration]
+order: 1
+---
+
+
+This document describes how to version your [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file and get started with bundling config.
+
+* TOC
+{:toc}
+
+## Getting Started with Config Bundling
+{:.no_toc}
+
+1. Add your project on the Add Projects page if it is a new project. For an existing Project, go to the Project Settings and enable [Build Processing]({{ site.baseurl }}/2.0/build-processing/) with the radio button.
+
+2. Install the CircleCI-Public CLI by following the [Using the CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/) documentation.
```

Comment:
This seems orthogonal to the reuse features -- I suggest we start encouraging anyone doing config to install the CLI to use `config validate`, but that's not unique to this particular set of features.

Reply:
I've found `circleci config process` to be very helpful in writing reusable config. It does what `circleci config validate` does, but also performs the substitution of references to commands, executors, and jobs.